[21452:18052:0314/234202.340:WARNING:chrome_main_delegate.cc(618)] This is Chrome version 111.0.5563.65 (not a warning)
[21452:13760:0314/234202.457:ERROR:model_type_store_service_impl.cc(38)] Failed to initialize ModelTypeStore backend: Init@components\sync\model\model_type_store_backend.cc:109: IO error: S:\PY_BOTVA\SELENIUM\BOTVA\Sync Data\LevelDB/MANIFEST-000001: Unable to create sequential file (ChromeMethodBFE: 7::NewSequentialFile::1)
[21452:14300:0314/234202.651:WARNING:leveldb_database.cc(123)] Unable to open S:\PY_BOTVA\SELENIUM\BOTVA\shared_proto_db\metadata: IO error: S:\PY_BOTVA\SELENIUM\BOTVA\shared_proto_db\metadata/MANIFEST-000001: Unable to create sequential file (ChromeMethodBFE: 7::NewSequentialFile::1)
[21452:14300:0314/234202.658:WARNING:leveldb_database.cc(123)] Unable to open S:\PY_BOTVA\SELENIUM\BOTVA\shared_proto_db\metadata: IO error: S:\PY_BOTVA\SELENIUM\BOTVA\shared_proto_db\metadata/MANIFEST-000001: Unable to create sequential file (ChromeMethodBFE: 7::NewSequentialFile::1)
[21452:14300:0314/234202.666:WARNING:leveldb_database.cc(123)] Unable to open S:\PY_BOTVA\SELENIUM\BOTVA\shared_proto_db\metadata: IO error: S:\PY_BOTVA\SELENIUM\BOTVA\shared_proto_db\metadata/MANIFEST-000001: Unable to create sequential file (ChromeMethodBFE: 7::NewSequentialFile::1)
[21452:14300:0314/234202.673:WARNING:leveldb_database.cc(123)] Unable to open S:\PY_BOTVA\SELENIUM\BOTVA\shared_proto_db\metadata: IO error: S:\PY_BOTVA\SELENIUM\BOTVA\shared_proto_db\metadata/MANIFEST-000001: Unable to create sequential file (ChromeMethodBFE: 7::NewSequentialFile::1)
[21452:14300:0314/234202.702:WARNING:value_store_frontend.cc(47)] Reading ahfgeienlihckogmohjhadlkjgocpleb.alarms from  failed: IO error: .../MANIFEST-000001: Unable to create sequential file (ChromeMethodBFE: 7::NewSequentialFile::1)
[21452:14300:0314/234202.708:WARNING:value_store_frontend.cc(47)] Reading mhjfbmdgcfjbbpaeojofohoefgiehjai.alarms from  failed: IO error: .../MANIFEST-000001: Unable to create sequential file (ChromeMethodBFE: 7::NewSequentialFile::1)
[21452:18052:0314/234203.180:INFO:CONSOLE(0)] "[DOM] Input elements should have autocomplete attributes (suggested: "current-password"): (More info: https://goo.gl/9p2vKq) %o", source: https://avatar.botva.ru/ (0)
[21452:18052:0314/234203.221:INFO:CONSOLE(558)] "Uncaught SyntaxError: Unexpected end of input", source: https://lbn2.ddestiny.ru/static/js/api/destiny_scope.js (558)
[21452:18052:0314/234205.757:INFO:CONSOLE(558)] "Uncaught SyntaxError: Unexpected end of input", source: https://lbn2.ddestiny.ru/static/js/api/destiny_scope.js (558)
[21452:18052:0314/234206.803:INFO:CONSOLE(558)] "Uncaught SyntaxError: Unexpected end of input", source: https://lbn2.ddestiny.ru/static/js/api/destiny_scope.js (558)
[21452:18052:0314/234209.276:INFO:CONSOLE(558)] "Uncaught SyntaxError: Unexpected end of input", source: https://lbn2.ddestiny.ru/static/js/api/destiny_scope.js (558)
[21452:18052:0314/234209.920:INFO:CONSOLE(558)] "Uncaught SyntaxError: Unexpected end of input", source: https://lbn2.ddestiny.ru/static/js/api/destiny_scope.js (558)
[21452:18052:0314/234211.992:ERROR:device_event_log_impl.cc(222)] [23:42:11.992] USB: usb_device_handle_win.cc:1046 Failed to read descriptor from node connection: ѕрисоединенное к системе устройство не работает. (0x1F)
[21452:18052:0314/234211.993:ERROR:device_event_log_impl.cc(222)] [23:42:11.993] USB: usb_device_handle_win.cc:1046 Failed to read descriptor from node connection: ѕрисоединенное к системе устройство не работает. (0x1F)
[21452:18052:0314/234218.233:WARNING:pref_notifier_impl.cc(41)] Pref observer for media_router.cast_allow_all_ips found at shutdown.
